src/jobs.rs

```diff
@@ -87,18 +121,29 @@ impl Job {
                 let slug = slug
                     .as_ref()
                     .ok_or_else(|| StatusCode::BAD_REQUEST.into_response())?;
-                Command::new(cmd)
-                    .arg("deploy")
-                    .arg(slug)
-                    .stdin(Stdio::null())
-                    .stdout(Stdio::piped())
-                    .stderr(Stdio::piped())
-                    .kill_on_drop(true)
-                    .spawn()
-                    .map_err(|e| {
-                        error!("failed to spawn process: {e}");
-                        StatusCode::INTERNAL_SERVER_ERROR.into_response()
-                    })?
+
+                match &toml {
+                    None => enarx_deploy(&cmd, slug)?,
+                    Some(toml) => {
+                        let slug = Slug::new(slug.clone())
+                            .ok_or_else(|| StatusCode::BAD_REQUEST.into_response())?;
+                        let wasm_bytes = slug
+                            .read("main.wasm")
+                            .await
+                            .map_err(|_| StatusCode::INTERNAL_SERVER_ERROR.into_response())?
```

Comment:
These errors will be impossible to debug in production settings. How about we don't discard the error, but rather log it?
Something like this:
```suggestion
                            .map_err(|e| {
                            	warn!("failed to fetch Wasm: {e}"); // or debug!, error!, whichever is appropriate
                            	StatusCode::INTERNAL_SERVER_ERROR.into_response()
                            })?
```